pipeline-model-definition/src/test/resources/options/restartableFromStageOptionSetFalse.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+pipeline {
```

Comment:
Are these files being used anywhere? Maybe you meant to add a test case to `BasicModelDefTest` (or a new class that extends `AbstractModelDefTest`) that runs these and verifies the behavior?

Reply:
Hi @dwnusbaum! I've update my description with question i wanted to ask. You are right. I wanted to add a test, but failed in loading the resources. Is there some kind of, let say, framework in place that i have to use, or just use standard Java way of loading resources?

Reply:
Take a look at tests in `BasicModelDefTest` (e.g. https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/2654ca721309dc727b2d7b76c4383724f599606d/pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java#L558) and `AbstractModelDefTest.expect`. You put your test resources in a specific folder (I do not recall offhand, so compare against other tests, and then call `expected('your_resource').go()` to create a `WorkflowJob` that uses that script and runs it, returning the `WorkflowRun`, and then you can assert things on the `WorkflowRun` as in your existing test.

Reply:
Thanks @dwnusbaum. I really appreciate your help. I'll work on this changes this week with all that you've mentioned in mind.